conans/test/integration/toolchains/microsoft/vcvars_test.py

```diff
@@ -126,3 +126,19 @@ class TestConan(ConanFile):
 
     vcvars = client.load("conanvcvars.bat")
     assert 'vcvarsall.bat"  amd64 8.1 -vcvars_ver=14.3' in vcvars
+
+@pytest.mark.skipif(platform.system() != "Windows", reason="Requires Windows")
+def test_deactivate_vcvars_message():
```

Comment:
Can we get a test under powershell too? Just to avoid any issues in the future :)

Reply:
Done